src/components/NotificationRow.tsx

```diff
@@ -35,6 +35,18 @@ export const NotificationRow: React.FC<IProps> = ({
       const url = generateGitHubWebUrl(notification.subject.url);
       shell.openExternal(url);
     }
+
```

Comment:
We know the first and third conditionals will always be true at this point of code execution, so we probably can clean this up a little bit:

```js
    // Some Notification types from GitHub are missing urls in their subjects.
    if (notification.subject.url) {
      const url = generateGitHubWebUrl(notification.subject.url);
      shell.openExternal(url);
    // For discussions, we can at least send users to the main discussions page.
    } else if (notification.subject.type === SubjectType.Discussion) {
      const url = generateGitHubWebUrl(`${notification.repository.url}/discussions`);
      shell.openExternal(url);
    }
```

Reply:
Done 😉